infra/terraform/external_function/snowalert.tf

```diff
@@ -2,37 +2,98 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "3.34.0"
     }
+    
+    # snowflake = {
+    #   source = "chanzuckerberg/snowflake"
+    #   version = "0.24.0"
+    # }
+  
   }
 }
 
 provider "aws" {
   region = "us-west-2"
 }
 
+
+
 module "snowflake_api_integration_aws_gateway" {
   source                            = "./snowalert"
   prefix                            = "snowalert"
-  snowflake_integration_external_id = var.snowflake_integration_external_id
+  # snowflake_integration_external_id = var.snowflake_integration_external_id

```

Comment:
Is this okay to remove?

Reply:
yes

Reply:
yes, now we are getting the external_id as an output direct from Snowflake_api_integration resourse so no need to set it via variable